festim/hydrogen_transport_problem.py

```diff
@@ -196,6 +196,10 @@ def define_temperature(self):
         elif callable(self.temperature):
             arguments = self.temperature.__code__.co_varnames
             if "t" in arguments and "x" not in arguments and "T" not in arguments:
```

Comment:
should take the chance to remove this too
```suggestion
            if "t" in arguments and "x" not in arguments
```

Reply:
Why? if `T` is in the arguments then it has to be a `Function` not a `Constant`

Reply:
This snippet is for defining the temperature, it was accidentally left in from a previous PR